torchtext/datasets/text_classification.py

```diff
@@ -113,7 +112,8 @@ def get_vocab(self):
         return self._vocab
 
 
-def _setup_datasets(dataset_name, root='.data', ngrams=2, vocab=None, include_unk=False):
+def _setup_datasets(dataset_name, tokenizer=get_tokenizer("basic_english"),
```

Comment:
A keyword argment such as 'root' can always be used positionally. So, if you're adding tokenizer a call such as ```AG_NEWS("/some/path", 2)``` is now invalid. To solve this you'll need need to move tokenizer to the very end.

Seeing this function call we should spend some time on making sure they align between this set of datasets and the unsupervised datasets we just added. I see that [over there](https://github.com/pytorch/text/blob/master/torchtext/datasets/language_modeling.py#L75) tokenizer comes first. So I'd suggest to move it at the end for unsupervised datasets as well.

Reply:
Further there is also a different set of options. It might make sense to break BC to replace ```include_unk``` with the more generic ```removed_tokens=[]```.

Reply:
Fixed.